dogu.json

```diff
@@ -18,7 +18,19 @@
     "Path": "/var/lib/nexus",
     "Owner": "1000",
     "Group": "1000"
-  }],
+  },
+    {
+      "Name": "backup",
+      "Path": "/var/lib/NexusBackup",
```

Comment:
I think the name backup, does not fit very well. I think we should use a name like migration or something similar. The name backup could also be conflicting with a real backup concept.